storybookjs/storybook
# Fix display of large components

Add CSS to allow for oversized components to scroll in all directions. See [issue #1225](https://github.com/storybooks/storybook/issues/1225#issuecomment-307259951).

Issue: Overflowing portions of large components are inaccessible when using Centered plugin.

## What I did

I added `overflow: auto` to the outer wrapper and `margin: auto` to a new inner wrapper.

## How to test

Create a component larger than your viewport and check that you can stroll both up and down to see the entire component.
